### PR TITLE
improve engrish

### PR DESCRIPTION
**Pull request by nicosuter**
made the english thiccer by applying an advanced grammar filter.

In all seriousness, I improved the English in the README a bit. Feel free to let me know if you'd like any changes, either via a comment on this PR or via Discord.

### CODE REVIEW

**Comment by nicosuter** [comment]
Question - would you prefer "Feel free to fork this repo and make any edits you think would be beneficial. An IsThicc staff member will review your pull request and approve or deny it." or
"Feel free to fork this repo and make any edits you think would be beneficial. After you create a pull request, an IsThicc staff member will review it."?

I'm personally more of a fan of the second version.

File: README.md [hunk 1]
```diff
@@ -11,12 +11,12 @@ The IsThicc-Bot is a small bot to help IsThicc staff with everyday tasks. Most b
 
 ## Contributing 
 
-Feel free to fork this repo and make any edits you think would be good. An IsThicc staff member will review your pull request and either approve or deny.
+Feel free to fork this repo and make any edits you think would be beneficial. An IsThicc staff member will review your pull request and approve or deny it.
 
 ### Please note:
 
-We reserve the right to reject any pull request or edits without reason or warning. When editing any IsThicc source code you waive all rights over the code you contribute. Any attempt to claim the code as your own will be met with consequences to our full extent.
+We reserve the right to reject any pull request or edits without reason or warning. When editing any IsThicc source code, you waive all rights over the code you contribute. Any attempt to claim the code as your own will be met with consequences to the fullest extent of the law.
```
Comment (Mrmagicpie) [inline README.md]: We generally do not allow the editing of our legal information. Everything else looks good in this commit but this gives a different context to the one we want.

**Review by Mrmagicpie — CHANGES_REQUESTED** [review]
Everything looks good, but since I'm bad at git, can you revert that one paragraph? Thanks for the pr, Ryder!

**Comment by nicosuter** [comment]
version 1, submitting second pr with alternative legal stuff

**Review by Mrmagicpie — APPROVED** [review]
Looks good. Thank you for your contribution.

**Pull request merged**